2021-04-20 09:24:03,006 INFO    MainThread:1139336 [wandb_setup.py:_flush():69] setting env: {}
2021-04-20 09:24:03,006 INFO    MainThread:1139336 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-20 09:24:03,006 INFO    MainThread:1139336 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-20 09:24:03,007 INFO    MainThread:1139336 [wandb_init.py:_log_setup():319] Logging user logs to /home/ubuntu/evgeny/GEP-forecasts/EVERGI/wandb/run-20210420_092403-3bw8bbsh/logs/debug.log
2021-04-20 09:24:03,007 INFO    MainThread:1139336 [wandb_init.py:_log_setup():320] Logging internal logs to /home/ubuntu/evgeny/GEP-forecasts/EVERGI/wandb/run-20210420_092403-3bw8bbsh/logs/debug-internal.log
2021-04-20 09:24:03,007 INFO    MainThread:1139336 [wandb_init.py:init():352] calling init triggers
2021-04-20 09:24:03,007 INFO    MainThread:1139336 [wandb_init.py:init():357] wandb.init called with sweep_config: {}
config: {'layers': 1, 'dropout': 0.3, 'neurons': 64, 'learning rate': 0.001, 'batch_size': 1500, 'architecture': 'local', 'dataset': 'Columbia', 'building': 'B1', 'epochs': 40, 'patience': 5}
2021-04-20 09:24:03,007 INFO    MainThread:1139336 [wandb_init.py:init():401] starting backend
2021-04-20 09:24:03,007 INFO    MainThread:1139336 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-20 09:24:03,018 INFO    MainThread:1139336 [backend.py:ensure_launched():123] starting backend process...
2021-04-20 09:24:03,030 INFO    MainThread:1139336 [backend.py:ensure_launched():127] started backend process with pid: 1139414
2021-04-20 09:24:03,032 INFO    MainThread:1139336 [wandb_init.py:init():406] backend started and connected
2021-04-20 09:24:03,035 INFO    MainThread:1139336 [wandb_init.py:init():446] updated telemetry
2021-04-20 09:24:03,036 INFO    MainThread:1139336 [wandb_init.py:init():465] communicating current version
2021-04-20 09:24:03,737 INFO    MainThread:1139336 [wandb_init.py:init():470] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-20 09:24:03,738 INFO    MainThread:1139336 [wandb_init.py:init():478] communicating run to backend with 30 second timeout
2021-04-20 09:24:03,990 INFO    MainThread:1139336 [wandb_init.py:init():503] starting run threads in backend
2021-04-20 09:24:04,545 WARNING MainThread:1139336 [wandb_init.py:init():748] interrupted
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/evgeny/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 728, in init
    run = wi.init()
  File "/home/ubuntu/anaconda3/envs/evgeny/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 506, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/ubuntu/anaconda3/envs/evgeny/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 746, in communicate_run_start
    result = self._communicate(rec)
  File "/home/ubuntu/anaconda3/envs/evgeny/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 525, in _communicate
    f = future.get(timeout)
  File "/home/ubuntu/anaconda3/envs/evgeny/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 81, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/ubuntu/anaconda3/envs/evgeny/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/ubuntu/anaconda3/envs/evgeny/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
